docs/source/runtime_troubles.rst

```diff
@@ -18,20 +18,32 @@ problems you may encounter.
 
 .. _string-literal-types:
 
-String literal types
---------------------
+String literal types and type comments
+--------------------------------------
+
+Mypy allows you to add type annotations using ``# type:`` type comments.
+For example:
+
+.. code-block:: python
+
+   a = 1  # type: int
+
+   def f(x):  # type: (int) -> int
```

Comment:
By the way, are multi-line function type comments documented somewhere, i.e. comments like these:
```py
def f(x,  # type: int
      y,  # type: str
):
    # type: (...) -> None
    ...

Reply:
Not any more, I think they were only documented in the Python 2 cheatsheet